MSAL/src/MSALErrorConverter.m

```diff
@@ -201,6 +212,7 @@ + (NSError *)errorWithDomain:(NSString *)domain
         msalUserInfo[mappedKey] = userInfo[key];
     }
 
+    if (!msalUserInfo[MSIDCorrelationIdKey] && correlationId) msalUserInfo[MSIDCorrelationIdKey] = correlationId;
```

Comment:
do we want to save it as a string and not UUID? if anyone tries to serialize error, UUID would fail I believe

Reply:
Just read the code a little bit and notice we are using string format already in msid. Will make the changes